package.json

```diff
@@ -43,6 +44,7 @@
     "@skpm/sketchapp-json-plugin": "^0.2.0",
     "airbnb-prop-types": "^2.10.0",
     "error-stack-parser": "^2.0.0",
+    "gitbook-plugin-github": "^2.0.0",
```

Comment:
That shouldn't be here

Reply:
Interestingly this dependency snuck in when I was running `yarn docs:watch`. Will remove it, however I can't guarantee it's not coming back some time in the future at someone else's place.

I think it's caused by this line in the `book.json`: https://github.com/airbnb/react-sketchapp/blob/master/book.json#L8

Reply:
fixed in https://github.com/airbnb/react-sketchapp/pull/422/commits/8758f3783130a9138b1e40f4bd8dbe0ef6fcd5a6